projects/swimlane/ngx-ui/src/lib/components/date-time/date-time.component.html

```diff
@@ -1,3 +1,47 @@
+<div class="ngx-date-time__container"
+  ngx-tooltip
```

Comment:
Do we want to set the tooltip on the container instead of on the actual input value? This will result in showing the tooltip on hovering anywhere in this component.

Reply:
<img width="1225" alt="Screen Shot 2022-01-03 at 9 03 25 AM" src="https://user-images.githubusercontent.com/12345416/147952494-628e678a-7c2e-471b-a248-19ef2c51bbcb.png">
We should hide the tooltip when there is no value